src/utils/indexer/getBlockTime.ts

```diff
@@ -1,49 +1,26 @@
-import { getSdk } from "../cache/getSdk";
-import { logger } from "../logger";
+import { eth_getBlockByNumber, getRpcClient } from "thirdweb";
+import { getChain } from "../chain";
+import { thirdwebClient } from "../sdk";
 
-const KNOWN_BLOCKTIME_SECONDS = {
-  1: 12,
-  137: 2,
-} as Record<number, number>;
+export const getBlockTimeSeconds = async (
```

Comment:
Simplifies logic + moves to v5.

Previously this code was summing the timestamps for 100 blocks. Now it just takes the diff between the n and n-100 block, which should be equivalent.